README.md

```diff
@@ -18,7 +18,26 @@ Stream allows developers to rapidly deploy scalable feeds and chat messaging wit
 
 **V4 Migration Guide**
 
-For upgrading from V3 to V4, please refer to the [V4 Migration Guide](https://getstream.io/chat/docs/sdk/flutter/guides/migration_guide_4_0/)
+> [!WARNING]
+> Flutter `> = v3.16.x` uses material 3 by default which breaks the UI of some of the components.
+> If you want to use these versions, you need to set the `useMaterial3` parameter to `false` 
```

Comment:
So if someone wants to use material 3 in his app he shouldn't use latest stable version of our SDK? Maybe we should at least note that we are working on new version that will work fine with material 3  - or don't we?